leelastar/network/layers.py

```diff
@@ -0,0 +1,79 @@
+import torch.nn as nn
+
+from collections import OrderedDict
+
+
+class SqueezeExcitation(nn.Module):
+    """Construct a squeeze excitation block for use in a residual block.
+
+    This uses the squeeze excitation block as proposed by `Hu et al. 2017`_.
+
+    .. _Hu et al. 2017:
+        https://arxiv.org/abs/1709.01507
+
+    """
+
+    def __init__(self, channels, ratio):
+        super().__init__()
+
+        self.global_pooling = nn.AdaptiveAvgPool2d(1)
+        self.dense_linear_1 = nn.Linear(channels, channels // ratio)
+        self.relu = nn.ReLU(inplace=True)
+        self.dense_linear_2 = nn.Linear(channels // ratio, 2 * channels)
+
+    def forward(self, x):
+        x_before = x
+        n, c, h, w = x.size()
```

Comment:
Can we use more clear variable names here?

Reply:
Going to do that.

Reply:
Just saw that as well, so its okay to leave it like that, but keep the unused ones `_`,  as recommended by pylint